src/usr/local/buildpack/tools/v2/kustomize.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+function install_tool () {
+  local versioned_tool_path
+  local file
+  local arch=linux_amd64
+
+  if [[ "$(uname -p)" = "aarch64" ]]; then
+    arch=linux_arm64
+  fi
+
+  local kustomize_file="${TOOL_NAME}_v${TOOL_VERSION}_${arch}.tar.gz"
```

Comment:
i think you need to split declaration and assignment, otherwise shellcheck checks will fail

Reply:
Hmm, I see multiple other usages like that, f.ex https://github.com/containerbase/base/blob/fc6b9dbdf86ceb8e05f16ad5100940ef9319a31c/src/usr/local/buildpack/tools/v2/maven.sh#L15

I can split them if needed ofc